known-issues.yml

```diff
@@ -730,6 +730,15 @@ zib-VisualFunction.VisualAid.Product:
     Device.udiCarrier:*UdiCarrier.carrierHRF:
       - datatype: string instead of CD compatible datatype
         reason: The mapping of ProductID is not straightforward and relates to multiple elements in FHIR. In result, the mapping of the concept to FHIR will not match on a datatype level.
+        
+zib-wounds.WoundImage:
+  zib deviations:
+    DocumentReference.content.attachment.url:
+      - alias: WondFoto instead of Wondfoto
+        for: NL-CM:19.4.11 (Burnwound.WoundImage)
+        reason: The zib concept name "Wondfoto" is incorrect and must be corrected to "WondFoto".
+      - datatype: url instead of ED
+        reason: to be filled.
```

Comment:
Don't forget to update this reason ;)

Reply:
In line with the update profiling guidelines, the mapping has been moved and this comment has been removed.